vocabularies/survey-type.ttl

```diff
@@ -241,25 +260,29 @@ svytp:radiometric a skos:Concept ;
         "Gamma-Ray Spectrography"@en ;
     skos:definition "A survey where the emissions of natural gamma-rays are recorded to determine the quantity of radioelements (Potassium, Uranium, Thorium)."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "rm" ;
     skos:prefLabel "Radiometric"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 
 svytp:spectral a skos:Concept ;
     skos:altLabel "hyperspectral"@en ;
     skos:definition "A survey capturing image information and intensity-per-wavelength-band to be used for spectral analysis e.g. ASTER, MODIS."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "sp" ;
     skos:prefLabel "Spectral"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 
 svytp:gravity a skos:Concept ;
     skos:definition "A survey where the variation of the Earth's gravity field in measured to determine the distribution of density in the region"@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "gv" ;
     skos:prefLabel "Gravity"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
 

```

Comment:
Should we include a new concept for "Spatial" surveys? 
Stuff like stereophotogrammetry, Shuttle Radar Topography Mapping, LIDAR, Theodolite surveying etc?
@dxwell @DavidCrosswellGSQ @KellyVance @geoderekh @Greenwoodmatthew 

e.g.
```
svytp:spatial a skos:Concept ;



    skos:definition "A survey measuring the spatial characteristics of an object or region."@en ;



    skos:inScheme <http://linked.data.gov.au/def/survey-type> ;



    skos:notation "st" ;



    skos:prefLabel "Spatial"@en ;



    skos:topConceptOf <http://linked.data.gov.au/def/survey-type> .
```

Reply:
I dont know if spatial is the right term. 'Elevation' survey 'digital terrain model' survey?

Reply:
@GSQ-AI Good idea. Added.

@Greenwoodmatthew I think 'spatial' can encompass any survey that defines the shape of stuff. So topographic surveys, point-cloud surveys (eg. ground surveys, underground mine, or object rendering). 

Things like Topographic could be a narrower terms if-and-when we need that granularity with Digital Elevation, Digital Terrain, Elevation as altLabels